### DISCUSSION TITLE
v2ray无法访问网页（任何网页，油管等都不行）

### DISCUSSION THREAD

**Discussion by kingsleyli920** [v2fly/v2ray-core#1139] [Q&A]
之前VPS过期忘记续费，但是在VPS被回收前顺利续费，并且VPS运行正常，可是之后就没法科学上网了。

后面申请新IP，改了端口，重装了系统，重新安装了v2ray，配置啥的看着都没问题。

VPS ping谷歌，百度没问题，本地ping VPS没问题，访问查看v2ray日志，打印 “accepted tcp:gateway.icloud.com.cn:443” 也都是类似这种，查看错误没有报错。

按照网上找的，查看服务器时间是local time，防火墙关了和开放端口都试过。

我是github上那个一键安装的v2ray，没找到config.json文件，所以只知道需要的配置，没查到所有配置，不知道有问题不。

求大神解答，搞了一整天，实在没办法了。

**Comment by okudayukiko** [comment]
1. Set correct time with `timedatectl`
2. Update your DNS record
3. Try VLESS
